server/tsconfig.json

```diff
@@ -17,7 +17,8 @@
     "noImplicitAny": false,
     "strictNullChecks": true,
     "suppressImplicitAnyIndexErrors": true,
-    "noUnusedLocals": false
+    "noUnusedLocals": false,
+    "downlevelIteration": true
```

Comment:
This needs to be true so that TypeScript properly transpiles `for... of...` loops that iterate over Maps.